documentation/API_documentation/Carrier_Billing_Payment_API.md

```diff
@@ -176,7 +176,7 @@ Following table defines API endpoints of exposed REST based for Carrier Billing
 
 | **Quering payment Resource /list information** |
 | --------------------------------------- |
-| **HTTP Request**<br> GET\<base-url>/payment/v0/payments/{paymentId}<br>**Query Parameters**<br> No query parameters are defined.<br>**Path Parameters**<br> endUserId: Identify the mobile account  charged - in general mobile number is provided<br> fields: Comma-separated attributes to be provided in response <br> offset: Requested index for start of resources to be provided in response <br> limit: Requested number of resources to be provided in response <br>**Request Body Parameters**<br> No request body parameters are defined.<br>**Response**<br><br> **200: a list of payment(s).**<br>  Response body:<br> An array of payment representation (could be a subset of a complete representation depending on fields filtering). <br>Note: It could be empty list if no payment match search criteria.<br><br> **401:** Un-authorized. <br> **403:** Forbidden. <br> <br> **503:** Service temporarily unavailable. |
+| **HTTP Request**<br> GET\<base-url>/payment/v0/payments/{paymentId}<br>**Query Parameters**<br> No query parameters are defined.<br>**Path Parameters**<br> endUserId: Identify the mobile account  charged - in general mobile number is provided<br> fields: Comma-separated pattributes to be provided in response <br> offset: Requested index for start of resources to be provided in response <br> limit: Requested number of resources to be provided in response <br>**Request Body Parameters**<br> No request body parameters are defined.<br>**Response**<br><br> **200: a list of payment(s).**<br>  Response body:<br> An array of payment representation. It could be empty if no payment match search criteria.<br><br> **401:** Un-authorized. <br> **403:** Forbidden. <br> <br> **503:** Service temporarily unavailable. |

```

Comment:
```suggestion
| **HTTP Request**<br> GET\<base-url>/payment/v0/payments/{paymentId}<br>**Query Parameters**<br> No query parameters are defined.<br>**Path Parameters**<br> endUserId: Identify the mobile account  charged - in general mobile number is provided<br> fields: Comma-separated attributes to be provided in response <br> offset: Requested index for start of resources to be provided in response <br> limit: Requested number of resources to be provided in response <br>**Request Body Parameters**<br> No request body parameters are defined.<br>**Response**<br><br> **200: a list of payment(s).**<br>  Response body:<br> An array of payment representation. It could be empty if no payment match search criteria.<br><br> **401:** Un-authorized. <br> **403:** Forbidden. <br> <br> **503:** Service temporarily unavailable. |

```